### ISSUE TITLE
Uncaught exception 'MoorContinueException' when Moor::routeNotFoundCallback()

### ISSUE THREAD

**Issue by xoan** [jeffturcotte/moor#15]
In a simple scenario:

```
# .htaccess
RewriteEngine On
RewriteRule ^.*$ bootstrap.php [QSA,NS]

<?php
// bootstrap.php
include '../libraries/moor/Moor.php';
Moor::route('/hello', function() {
    echo 'Hello!';
});
Moor::run();
```

When I try to get / (empty request_uri):

```
Fatal error: Uncaught exception 'MoorContinueException' in /srv/http/cineforiano/libraries/moor/Moor.php:611
Stack trace:
#0 /srv/http/cineforiano/libraries/moor/Moor.php(1241): Moor::triggerContinue()
#1 /srv/http/cineforiano/libraries/moor/Moor.php(762): Moor::validateMethodCallback('Moor::routeNotF...')
#2 /srv/http/cineforiano/libraries/moor/Moor.php(565): Moor::dispatchRoute(Object(stdClass))
#3 /srv/http/cineforiano/public/bootstrap.php(7): Moor::run()
#4 {main}
  thrown in /srv/http/cineforiano/libraries/moor/Moor.php on line 611
```

Instead to get a Moor::routeNotFoundCallback()

```
<h1>NOT FOUND</h1>
```

If I define a notFoundCallback() and assign it to Moor with setNotFoundCallback(), it works as expected.

```
<?php
// bootstrap.php
include '../libraries/moor/Moor.php';
function notFound() {
    echo 'Not Found!';
}
Moor::setNotFoundCallback('notFound');
Moor::route('/hello', function() {
    echo 'Hello!';
});
Moor::run();
```

**Comment by gabrielu** [comment]
Thanks for posting @xoan

**Comment by mattsah** [comment]
this is due to the default not found callback not being valid 1) its not public 2) it is on Moor which doe not extend MoorAbstractController - it is fixed in the dotink fork if you would like to try that.

**Comment by jeffturcotte** [comment]
Fixed in 589003e8bcb83f6b999955c60c2a840527b444e2. Thanks.
